freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/repair/changeprimarygw/ChangePrimaryGatewayService.java

```diff
@@ -62,11 +62,11 @@ public String selectNewPrimaryGatewayInstanceId(Stack stack, List<String> repair
                 }
             }
         } catch (NotFoundException e) {
-            LOGGER.debug("No primary gateway found, searching for an different primary gateway");
+            LOGGER.debug("No primary gateway found, searching for a different primary gateway");
             newPrimaryGatewayInstanceId = assignNewPrimaryGatewayInstanceId(stack, repairInstanceIds);
         } catch (CloudbreakOrchestratorException e) {
-            LOGGER.debug("The orchestrator failed to identify the primary gateway node", e);
-            throw new NotFoundException("No FreeIPA primary gateway could be selected", e);
+            LOGGER.debug("The orchestrator failed to identify the primary gateway node, searching for a different primary gateway", e);
+            newPrimaryGatewayInstanceId = assignNewPrimaryGatewayInstanceId(stack, repairInstanceIds);
```

Comment:
same question as above.

Reply:
what is the scenario difference between NotFoundException and CloudbreakOrchestratorException? 

catch(NotFoundException | CloudbreakOrchestratorException e) {
and just print the internal exception msg?

Reply:
I was considering combining these before. I will do that now.

For reference:`NotFoundException` is thrown when the FreeIPA primary gateway node has been deleted. While `CloudbreakOrchestratorException` is thrown when the FreeIPA primary gateway node still exists but there is a failure to communicate with it (e.g. stopped, unreachable, unhealthy, etc.).